app/controllers/api/likes_controller.rb

```diff
@@ -1,15 +1,25 @@
 class Api::LikesController < ApplicationController
+  before_action :find_like, only: [:destroy] 
+
   def create
     if already_liked?
-      render json: { message: "You can't like more than once" }, status: 422
+      destroy
     else
-    like = current_user.likes.create(like_params)
-    render status: 201 if like.persisted?
+      like = current_user.likes.create(like_params)
+      render status: 201 
```

Comment:
Why did you delete if like.persisted? :)

Reply:
Sorry this was a mistake, it's now back in business!

Reply:
I still can't see it...:(

Reply:
me too :((

Reply:
Now then..?

Reply:
Now I can see it!